_pkgdown.yaml

```diff
@@ -68,3 +68,8 @@ reference:
       - ExpHazOS
       - avgHRIntegExpOS
       - avgHRExpOS
+  - title: Empirical power
+    contents:
+      - logRankTest
+      - passedLogRank
```

Comment:
Question is if we want to export this at all - it might be better maybe here to not export this function, instead add `@keywords internal` and hence omit from this list here